copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -4,12 +4,51 @@
 Rotor Speed Control Setup
 =========================
 
-Rotor speed controller set up is described in video 3
-for electric powered helicopters and video 4 for piston 
-powered helicopters on the following page.
+The rotor speed control uses the throttle output to the engine to control rotor speed. 
+This can be done in a few different ways either through an open loop control called a 
+throttle curve, a set point signal to an external governor or electronic speed controller,
+or a closed loop internal governor. The RSC also controls the throttle for ground idle, 
+rotor run up and shut down. This wiki covers the set up of each of the parameters in the RSC.
 
-:ref:`traditional-helicopter-configuration`
+RSC mode
+========
 
+First, set the :ref:`RSC Mode <H_RSC_MODE>` parameter. The RSC modes are listed below with a short description. 
```

Comment:
H_RSC_MODE<H_RSC_MODE>    not   RSC_MODE<RSC_MODE>

should be changed everywhere, for all param links...thaks